Detectors/TRD/workflow/src/TRDDigitizerSpec.cxx

```diff
@@ -99,8 +99,11 @@ class TRDDPLDigitizerTask : public o2::base::BaseDPLDigitizer
         LOG(INFO) << "For collision " << collID << " eventID " << part.entryID << " found TRD " << hits.size() << " hits ";
 
         mDigitizer.process(hits, digits, labels);
+        mDigitizer.flush(digits, labels);
```

Comment:
You can't force flushing the digits here; this will break embedding.

Reply:
ja that is a carry over from me buggering around

Reply:
I thought I reverted this back to the other one late last night